src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,16 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Drawers {
+  export class ErrorBar extends Drawer {
```

Comment:
Oof this is gonna be tricky.  This is not really an `ErrorBar` Drawer but instead a ... `Line` Drawer...

Try to name it to something that is not specific to `ErrorBar`s and more aligned with `Line` that doesn't collide with `LineDrawer`